astropy/modeling/fitting.py

```diff
@@ -1181,81 +1297,168 @@ def __call__(self, model, x, y, z=None, weights=None,
                           "fit_info['message'] for more information.",
                           AstropyUserWarning)
 
-        # now try to compute the true covariance matrix
-        if (len(y) > len(init_values)) and cov_x is not None:
-            sum_sqrs = np.sum(self.objective_function(fitparams, *farg)**2)
-            dof = len(y) - len(init_values)
-            self.fit_info['param_cov'] = cov_x * sum_sqrs / dof
-        else:
-            self.fit_info['param_cov'] = None
+        return init_values, fitparams, cov_x
 
-        if self._calc_uncertainties is True:
-            if self.fit_info['param_cov'] is not None:
-                self._add_fitting_uncertainties(model_copy,
-                                               self.fit_info['param_cov'])
 
-        model_copy.sync_constraints = True
-        return model_copy
+class _NLLSQFitter(_NonLinearLSQFitter):
```

Comment:
Is this docstring even exposed in API doc, given it is a "hidden" class?

Reply:
No, but @nden originally requested it. The docstring does help document what is going on.